test/test_gnosis.js

```diff
@@ -92,8 +92,8 @@ describe('Gnosis', function () {
 
         await gnosis.setWeb3Provider(ganache.provider({ network_id: 4 }))
 
-        assert.equal(gnosis.olympiaToken.address, '0xa0c107db0e9194c18359d3265289239453b56cf2')
-        assert.equal(gnosis.olympiaAddressRegistry.address, '0x79da1c9ef6bf6bc64e66f8abffddc1a093e50f13')
+        assert.equal(gnosis.olympiaToken.address, '0x979861df79c7408553aaf20c01cfb3f81ccf9341')
```

Comment:
I know these changed upstream... I can't remember if that was intentional :sweat_smile: